pkg/util/k8sutil/self_hosted.go

```diff
@@ -141,14 +141,13 @@ done
 		Spec: v1.PodSpec{
 			// Self-hosted etcd pod need to endure node restart.
 			// If we set it to Never, the pod won't restart. If etcd won't come up, nor does other k8s API components.
-			RestartPolicy:                v1.RestartPolicyAlways,
-			Containers:                   []v1.Container{c},
-			Volumes:                      volumes,
-			HostNetwork:                  true,
-			DNSPolicy:                    v1.DNSClusterFirstWithHostNet,
-			Hostname:                     m.Name,
-			Subdomain:                    clusterName,
-			AutomountServiceAccountToken: func(b bool) *bool { return &b }(false),
```

Comment:
revert this. It is not related to this PR.

Reply:
I need to delete `AutomountServiceAccountToken `. That is basically all the change here.

Reply:
> I need to delete AutomountServiceAccountToken

What's the reason for it?

Reply:
Because we want to 
> Can you remove the changes related to AutomountServiceAccountToken?
This PR should be about PersistentVolumeClaimSpec.

The comment from your previous review.

Reply:
Can you explain why AutomountServiceAccountToken is related to PV?

Reply:
I just get it back. 

I deleted it while I was trying to remove changes to AutomountServiceAccountToken. Apparently, it shouldn't be removed here.